pkg/operation/care/constraints.go

```diff
@@ -171,9 +176,16 @@ func (c *Constraint) constraintsChecks(
 		maintenancePreconditionsSatisfiedConstraint = v1beta1helper.UpdatedConditionWithClock(c.clock, maintenancePreconditionsSatisfiedConstraint, status, reason, message, errorCodes...)
 	}
 
+	status, reason, message, err = c.checkIfCRDsWithConversionWebhooksPresent(ctx)
+	if err != nil {
```

Comment:
if condition can be inlined here.

Reply:
For all the other checks, it's done this way, so I followed the same to be consistent.